src/features/pass-survey/ui/ActivityItem.tsx

```diff
@@ -90,6 +90,12 @@ function ActivityItem({
     }
   }
 
+  function moveToNextItemWithDelay() {
+    if (!pipelineItem.additionalText?.required) {
+      setTimeout(() => setImmediate(() => next(true)), 200);
```

Comment:
Please come up with a different solution
Maybe we can use https://reactnative.dev/docs/interactionmanager#runafterinteractions